packages/react/src/combobox/root/ComboboxRoot.tsx

```diff
@@ -62,6 +65,21 @@ type ComboboxRootBaseProps<Value, Multiple extends boolean | undefined> = Omit<
    * @default false
    */
   multiple?: Multiple;
+  /**
+   * Controls whether the first matching item is highlighted automatically while filtering.
```

Comment:
To align with other boolean fields.
```suggestion
   * Whether the first matching item is highlighted automatically while filtering.
```